### ISSUE TITLE
runme_poe.py and runme.py does not handle restarts

### ISSUE THREAD

**Issue by trje3733** [radical-experiments/adaptive-bms-experiments#2]
runme_poe.py and runme.py do not contain the logic necessary to properly handle a restart. This logic is present in the old repo (https://github.com/radical-collaboration/expanded-ensemble/blob/0790df7b92d676cd28a4e3c14578326687887478/entk_scripts/runme.py#L85). Restarts have become necessary to complete experiments because superMIC cancels jobs during periods of low node usage (I believe during the analysis stages).

**Comment by vivek-bala** [comment]
That's correct. Do you want to take a crack at adding the restart capability to runme_poe?

Also, did the idea of multiple analysis tasks or dummy analysis tasks work to avoid low node usage?

**Comment by trje3733** [comment]
I'll try to add the necessary logic. Initially I wasn't aware this section of the code changed so I was looking elsewhere for the issue. I'll keep this issue updated on progress.
